frontend/src/services/AttributesService.js

```diff
@@ -371,8 +371,6 @@ export const onChangeCompositionShared = (match, item) => {
         item.composition.intermediateObjectId = intermediateObjectId;
         item.composition.intermediateObjectName = intermediateObjectName;
         //copy some values into the profile_composition data record
-        item.browseName = match.browseName; //this becomes critical for adding on server side in new scenario

```

Comment:
Need to make sure this doesn't regress anything.

Reply:
See other comment: https://github.com/cesmii/ProfileDesigner/pull/93#discussion_r1206062384. This is indeed no longer necessary.